x-pack/plugins/fleet/public/components/multiple_agent_policy_summary_line.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiBadge,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiPopoverFooter,
+  EuiButton,
+  EuiListGroup,
+  type EuiListGroupItemProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { CSSProperties } from 'react';
+import { useMemo } from 'react';
+import React, { memo, useState } from 'react';
+
+import type { AgentPolicy } from '../../common/types';
+import { useLink } from '../hooks';
+const MIN_WIDTH: CSSProperties = { minWidth: 0 };
+
+export const MultipleAgentPoliciesSummaryLine = memo<{
+  policies: AgentPolicy[];
+  direction?: 'column' | 'row';
+}>(({ policies, direction = 'row' }) => {
+  const { getHref } = useLink();
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  // as default, show only the first policy
+  const policy = policies[0];
```

Comment:
does `policies` always have at least 1 length? if so, the check in line 81 is unnecessary

Reply:
Yes, in fact I added it when I wrote the tests. It could be removed but if the component gets used somewhere else with empty policies it will be there.